src/components/ExternalLink/_ExternalLink.scss

```diff
@@ -20,3 +23,10 @@
     }
   }
 }
+
+.coa-ExternalLinkMaterial {
+  padding: 0 3px;
+  font-size: 16px;
+  position: relative;
+  top: 3px;
+}
```

Comment:
The class `coa-ExternalLink ` is modified so that it allows for opacity and capitaliation as specified in the issue ![#2011](https://app.zenhub.com/workspaces/office-of-design-and-delivery-5a78b88e1ce69f3510b678ef/issues/cityofaustin/techstack/2011). The class `coa-ExternalLinkMaterial ` sets the style for the icon.